### PR TITLE
Fix: update gitignore file

### PR DESCRIPTION
**Pull request by Sotatek-HauTran3**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -41,31 +28,6 @@ bld/
...
-BenchmarkDotNet.Artifacts/
-
-# .NET Core
-project.lock.json
-project.fragment.lock.json
-artifacts/
-
-# StyleCop
-StyleCopReport.xml
-
```
Comment (kjeom) [inline .gitignore:68]: @Sotatek-HauTran3 I think you have deleted Windows platform related files. Is it okay when running in windows?

File: .gitignore [hunk 2]
```diff
@@ -3,19 +3,6 @@
 ##
 ## Get latest from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore
```
Comment (kjeom) [inline .gitignore:4]: @Sotatek-HauTran3 Please remove it because we deleted contents visual studio automatically generate

File: .gitignore [hunk 3]
```diff
@@ -41,31 +28,6 @@ bld/
 # Visual Studio 2017 auto generated files
 Generated\ Files/
```
Comment (kjeom) [inline .gitignore:29]: It is also Visual Studio related content

Comment (Sotatek-HauTran3) [reply .gitignore:29]: yes, i should be here, because of development ide

File: .gitignore (hunk 1 above)
Comment (Sotatek-HauTran3) [reply .gitignore:68]: It's ok

Comment (kjeom) [reply .gitignore:68]: It is also visual studio related [contents](https://github.com/klaytn/web3klaytn/pull/362/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947L69)

**Comment by kjeom** [comment]
I will fix it again after SDKs(ethers, web3j, web3py, web3js) are merged.

**Review by kjeom — APPROVED** [review]

**Pull request merged**